### DISCUSSION TITLE
How to use two websites in Cors?

### DISCUSSION THREAD

**Discussion by kevinmamaqi** [socketio/socket.io#3805] [Q&A]
Currently I am using socket.io with node with the followin setup for CORS:

```
const io = socketio(server, {
	cors: {
		origin: ['https://mydomain1.com', 'http://localhost.com'],
		methods: ["GET", "POST"],
	},
});
```

But I always get an error: **_has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header contains multiple values_**

Is there a way to allow multiple domains?

**Comment by darrachequesne** [comment]
I was not able to reproduce: https://github.com/socketio/socket.io-fiddle/tree/cors

Is there anything in your setup that could add another "Access-Control-Allow-Origin" header?
